sqlglot/dialects/hive.py

```diff
@@ -284,6 +276,18 @@ class Generator(Generator):
             exp.UnixToTimeStr: rename_func("FROM_UNIXTIME"),
         }
 
+        WITH_PROPERTIES = {exp.AnonymousProperty}
+
+        TABLE_PROPERTIES = {
```

Comment:
why split it up into table_properties and udf_properties? do you distinguish that somehow? doesn't seem to be a difference with root properties

Reply:
I don't have a strong justification for this, I'll change it back.